TKGame/UI/GameMenu.cs

```diff
@@ -1,15 +1,183 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.Linq;
+using Myra.Graphics2D.Brushes;
 using Myra.Graphics2D.UI;
+using Microsoft.Xna.Framework;
+using Myra.Graphics2D.TextureAtlases;
+using TKGame.BackEnd;
+using FontStashSharp;
+using System.IO;
+using Microsoft.Xna.Framework.Graphics;
+using TKGame.Players;
 
 namespace TKGame.UI
 {
     public class GameMenu : IMenu
     {
-        // TODO: Implement gameplay menu
-        public IMultipleItemsContainer Container => throw new NotImplementedException();
+        public IMultipleItemsContainer Container { get { return grid; } }
+        private Grid grid;
+        private DebugMenu debugMenu;
+        private Panel panel;
+        private HorizontalStackPanel panelHsp;
+        private FontSystem fontSystem;
+        private Label playerHealthLabel;
+        private bool playerGotPowerup;
+
+		private readonly int numCols = 1;
+        private readonly int numRows = 2;
+        private readonly int fontSize = 24;
+        private readonly int playerHpFontSize = 48;
+        private readonly string weaponImageId = "weapon";
+
+
+        public GameMenu()
+        {
+            Initialize();
+            LoadContent();
+        }
+
+        private void Initialize()
+        {
+            grid = new Grid();
+            debugMenu = new DebugMenu();
+            panel = new Panel();
+            panelHsp = new HorizontalStackPanel();
+            playerHealthLabel = new Label();
+            playerGotPowerup = false;
+
+			byte[] ttfData = File.ReadAllBytes(@"Content/Fonts/Retro Gaming.ttf");
+			fontSystem = new FontSystem();
+			fontSystem.AddFont(ttfData);
+		}
+
+        private void LoadContent()
+        {
+            for (int i = 0; i < numCols; i++)
+            {
+                grid.RowsProportions.Add(new Proportion(ProportionType.Auto));
+            }
+            
+            for (int i = 0; i < numRows; i++)
+            {
+                grid.ColumnsProportions.Add(new Proportion(ProportionType.Auto));
+            }
+
+            panel.GridRow = 0;
+            panel.GridColumn = 0;
+            panel.Height = 125;
+            panel.Width = TKGame.ScreenWidth;
+            panel.Margin = new Myra.Graphics2D.Thickness(100, 50, 0, 0);
+            grid.Widgets.Add(panel);
+
+            panelHsp.Spacing = 10;
+            panel.Widgets.Add(panelHsp);
+
+            playerHealthLabel.Font = fontSystem.GetFont(playerHpFontSize);
+            AddWidgetToHeaderPanel(playerHealthLabel, "HP");
+            
+            AddWidgetToHeaderPanel(CreateImageWidget(Art.WeaponTexture, 60, 60, weaponImageId), "Weapon");
```

Comment:
Probably want this to reference the Player's instance weapon so it can change if they player changes weapon somehow